src/api.py

```diff
@@ -951,6 +954,79 @@ def HandleGetMessageDataByDestinationHash(self, params):
         data += ']}'
         return data
 
+    def HandleDisseminateRawObject(self, params):
+        if len(params) != 1:
+            raise APIError(0, 'I need 1 parameter!')
+        payload = self._decode(params[0], "hex")
+
+        if protocol.checkAndShareObjectWithPeers(payload) == 0:
```

Comment:
This function is considered obsolete, but unfortunately there is no good replacement. Please put FIXME in front of it saying that it should be refactored in the future.